code/go/0chain.net/rest/restinterface/rest.go

```diff
@@ -0,0 +1,51 @@
+package restinterface
```

Comment:
Please do not use the `restinterface` package name, I think this is designed to break the dependencies cycle issue, but we'd better have a better name other than `*interface`. We actually don't need this package, and we can put the interface in the `rest` package, and have the callers to call `rest.SetupHandlers()` to register themselves to the `rest` package, or pass the handlers through args to the `restSetupHandlers() in top main function explicitly. All could break the depenence issue.

Reply:
I used the `restinterface` name to mirror the `chaincore/smartcontractinterface` package name. But yes we can merge rest and restinterface with some resturcturing.